packages/palette-docs/package.json

```diff
@@ -98,7 +98,7 @@
     "gatsby-plugin-s3": "^0.3.2",
     "gatsby-plugin-sharp": "^3.11.0",
     "gatsby-plugin-sitemap": "^4.7.0",
-    "gatsby-plugin-styled-components": "^4.11.0",
```

Comment:
This looks like a semver fail: 4.0 = Gatsby 3 (version we're on); 4.11 = Gatsby 4.